RInAppMessaging/Classes/UserDataCache.swift

```diff
@@ -2,6 +2,7 @@ internal protocol UserDataCacheable: AnyObject {
     func getUserData(identifiers: [UserIdentifier]) -> UserDataCacheContainer?
     func cacheCampaignData(_ data: [Campaign], userIdentifiers: [UserIdentifier])
     func cacheDisplayPermissionData(_ data: DisplayPermissionResponse, campaignID: String, userIdentifiers: [UserIdentifier])
+    func deleteUserData(identifiers: [UserIdentifier])
```

Comment:
Where is the separate anonymous user cache that was mentioned in the PR description?

Reply:
In CampaignRepository `lastUser`